Dockerfile

```
@@ -1,32 +1,38 @@
-FROM jupyter/base-notebook:python-3.7.6
-
+FROM jupyter/base-notebook
 
 USER root
 
-RUN apt-get -y update \
- && apt-get install -y dbus-x11 \
-   firefox \
-   xfce4 \
-   xfce4-panel \
-   xfce4-session \
-   xfce4-settings \
-   xorg \
-   xubuntu-icon-theme
+RUN apt-get -y update > /dev/null \
+ && apt-get -y upgrade > /dev/null \
```

```suggestion
RUN apt-get -y -q update \
 && apt-get -y -q upgrade \
```
I think it's useful to see what's going on, especially as upgrading packages inside a Docker image can brreak some assumptions.

---

Okay deal, note though that I reacted like you initially, but then when working in yuvipanda/jupyterhub-ssh I started to enjoy not being flooded with logs but only seeing the errors if there were errors.

Let's go back to not discarding things though.